site/source/user-manual/connecting/connecting-tor/tor-firefox/torff-android.rst

```diff
@@ -7,45 +7,50 @@ Configuring Firefox for Tor on Android
 .. caution::
   This guide assumes you have completed :ref:`setting up Tor for Android<tor-android>`. Please visit this section before proceeding as it is required for Firefox to properly work with Tor.
 
+.. note:: Setups may strongly vary across different Android forks. We have made specific mention of GrapheneOS which has been tested on Pixel 5 and Pixel 6.
+
 Once Tor is setup on your system, you can proceed to setup Firefox:
 
-1. Download `Firefox Beta <https://play.google.com/store/apps/details?id=org.mozilla.firefox_beta>`_ from the Play Store, or `Fennec <https://f-droid.org/en/packages/org.mozilla.fennec_fdroid/>`_ from F-Droid.
+#. Download `Firefox Beta <https://play.google.com/store/apps/details?id=org.mozilla.firefox_beta>`_ from the Play Store, or `Fennec <https://f-droid.org/en/packages/org.mozilla.fennec_fdroid/>`_ from F-Droid.
```

Comment:
Let's eliminate Firefox Beta and just recommend Fennec - Beta does not allow about:config on Calyx